### PR TITLE
change minSDK to 16, compile-, build- and targetSdkVersion to 22

### PR DESCRIPTION
**Pull request by Manu91**
change minSDK to 16, compile-, build- and targetSdkVersion to 21 and removed the @deprecated lines.

Green run:
https://jenkins.catrob.at/view/All-Categories/view/Catroid-multi-job/job/Catroid-Multi-Job-Custom-Branch-RELOADED/1325/

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -96,11 +96,11 @@ def getCurrentGitBranch() {
 
 
 android {
-    compileSdkVersion 19
-    buildToolsVersion '19.1.0'
+    compileSdkVersion 21
+    buildToolsVersion '21.1.2'
```
Comment (aried3r) [inline build.gradle]: Latest build tools are `22.0.1`.

File: build.gradle [hunk 2]
```diff
@@ -96,11 +96,11 @@ def getCurrentGitBranch() {
 
 
 android {
-    compileSdkVersion 19
-    buildToolsVersion '19.1.0'
+    compileSdkVersion 21
```
Comment (aried3r) [inline build.gradle]: Latest SDK version is 22.

File: build.gradle [hunk 3]
```diff
@@ -96,11 +96,11 @@ def getCurrentGitBranch() {
...
+    buildToolsVersion '21.1.2'
     defaultConfig {
-        minSdkVersion 10
-        targetSdkVersion 19
+        minSdkVersion 16
+        targetSdkVersion 21
```
Comment (aried3r) [inline build.gradle]: Same here, 22.

**Comment by avihric** [comment]
ok to test

**Comment by Manu91** [comment]
ok to test

**Comment by Manu91** [comment]
ok to test

**Comment by Manu91** [comment]
ok to test

**Comment by Manu91** [comment]
ok to test

**Comment by Manu91** [comment]
ok to test

**Comment by Manu91** [comment]
ok to test

**Pull request merged**